packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -192,6 +201,8 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
       content: false,
     }),
     activeSelectedIndex: PropTypes.number,
+    clearable: PropTypes.bool,
```

Comment:
I don't like that we have two new props in the API, but I don't have any better proposal for now...